### PR TITLE
Resolving bugs in text regex mapping and wso2 receiver viewing with custom mapping

### PR DESCRIPTION
**Pull request by ramindu90**

### CODE REVIEW

File: components/event-receiver/org.wso2.carbon.event.receiver.ui/src/main/resources/web/eventreceiver/eventReceiver_details.jsp [hunk 1]
```diff
@@ -291,7 +321,10 @@
                         </thead>
                         <tbody>
                         <%
-                            for (EventMappingPropertyDto metaEbProperties : eventReceiverConfigurationDto.getMetaMappingPropertyDtos()) {
+                            EventMappingPropertyDto[] metaMappingDto = eventReceiverConfigurationDto.getMetaMappingPropertyDtos();
+                            if(metaMappingDto != null && metaMappingDto.length > 0){
```
Comment (prabathabey) [inline components/event-receiver/org.wso2.carbon.event.receiver.ui/src/main/resources/web/eventreceiver/eventReceiver_details.jsp]: Formatting issues.

File: components/event-receiver/org.wso2.carbon.event.receiver.ui/src/main/resources/web/eventreceiver/eventReceiver_details.jsp [hunk 2]
```diff
@@ -322,10 +358,13 @@
                             </td>
                         </tr>
                         <%
+                                }
                             }
                         %>
                         <%
-                            for (EventMappingPropertyDto payloadEbProperties : eventReceiverConfigurationDto.getMappingPropertyDtos()) {
+                            EventMappingPropertyDto[] payLoadMappingDto = eventReceiverConfigurationDto.getMappingPropertyDtos();
+                            if(payLoadMappingDto != null && payLoadMappingDto.length > 0){
```
Comment (prabathabey) [inline components/event-receiver/org.wso2.carbon.event.receiver.ui/src/main/resources/web/eventreceiver/eventReceiver_details.jsp]: More formatting issues.

File: components/event-receiver/org.wso2.carbon.event.receiver.ui/src/main/resources/web/eventreceiver/eventReceiver_details.jsp (hunk 1 above)
Comment (ramindu90) [inline components/event-receiver/org.wso2.carbon.event.receiver.ui/src/main/resources/web/eventreceiver/eventReceiver_details.jsp]: Thanks @prabathabey, Changes are done in pull request - https://github.com/wso2/carbon-analytics-common/pull/122/

File: components/event-receiver/org.wso2.carbon.event.receiver.ui/src/main/resources/web/eventreceiver/eventReceiver_details.jsp (hunk 2 above)
Comment (ramindu90) [inline components/event-receiver/org.wso2.carbon.event.receiver.ui/src/main/resources/web/eventreceiver/eventReceiver_details.jsp]: Thanks @prabathabey, Changes are done in pull request - https://github.com/wso2/carbon-analytics-common/pull/122/

**Pull request merged**